webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala

```diff
@@ -254,45 +324,56 @@ class NodeGroupForm(
   }
 
   private[this] def onSubmit() : JsCmd = {
```

Comment:
Maybe we can hide the submit button for RuleTargets ?

Reply:
It his hidden in the `showFormTarget` method, here it's just a security if for whatever reason the js is triggered. Given the code structure, I wasn't ablt to make that state unrepresentable.